custom_components/kamstrup_403/pykamstrup/kamstrup.py

```diff
@@ -23,7 +23,7 @@ class Kamstrup:
 
     def __init__(self, serial_port: str, baudrate: int, timeout: float):
         """Initialize"""
-        self.ser = serial.Serial(port=serial_port, baudrate=baudrate, timeout=timeout)
+        self.ser = serial.serial_for_url("socket://irreader1.localdomain:3434", baudrate=baudrate, timeout=timeout)
```

Comment:
I do think the change should be like this:
```suggestion
        self.ser = serial.serial_for_url(url=serial_port, baudrate=baudrate, timeout=timeout)
```
What do you think?